devtools-ci-index.yaml

```diff
@@ -2186,6 +2186,16 @@
             ci_project: 'devtools'
             ci_cmd: 'cd ee_tests/ && /bin/bash cico_build_deploy.sh'
             timeout: '30m'
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
+            test_url: openshift.io
+            test_suite: logintest
+            cluster: cluster-one
+            after: devtools-fabric8-test-build-master
```

Comment:
note that the build-master pushes to prod-preview not to the prod clusters. Are you sure you want something there ?

Reply:
I was not aware of that - our use of the build-master was intended to build the docker images used by the tests - to enable the tests themselves to run quicker. Where is the reference to prod-preview configured? Thx!